src/components/Introduction.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import styled, {keyframes} from 'styled-components';
+import Photo from "../images/pic-01.png";
+import {Row} from 'react-bootstrap';
+import { ChevronDoubleDown } from "react-bootstrap-icons";
+import { Link } from "react-scroll"
+
+const ServiceWrap = styled.section`
+  overflow: hidden;
+  position: relative;
+  padding: 50px 0;
+`
+
+const CompanyImages = styled.div`
+  position: relative;
+  min-height: 1px;
+  padding: 0 15px;
+
+  img {
+    width: 100%;
+  }
+`
+const CompanyText = styled.div`
+  position: relative;
+  min-height: 1px;
+  padding-left: 15px;
+  padding-right: 15px;
+  padding: 12% 0 0;
+  text-align: center;
+
+  h3 {
+    color: #69696b;
+    font-weight: bold;
+    margin: 30px 0;
+    font-size: 1.5em;
+  }
+`
+const TextSalween = styled.div`
+  position: relative;
+  margin-bottom: 50px;
+  padding: 30px 0;
+
+  &:before {
+    content: "";
+    background: #aaa;
+    width: 200px;
+    height: 1px;
+    position: absolute;
+    left: 50%;
+    margin-right: -50%;
+    bottom: 0;
+    transform: translate(-50%, -50%);
+  }
+  h2 {
+    color: #f38227;
+    text-align: center;
+    margin: 0 0 30px;
+    font-size: 2.5em;
+  }
+  p {
+    font-size: 1.3em;
+        color: #69696B;
+    }
+    span {
+        color: #F38227;
+    }
+  }
+`
+const animation = keyframes`
```

Comment:
Could you specify what animation type ? for example `FadeInAnimation`, `EaseOutAnimation` etc...